openassessment/templates/openassessmentblock/edit/oa_edit.html

```diff
@@ -65,6 +65,22 @@ <h2 class="openassessment_alert_title">{% trans "Rubric Change Impacts Settings
                         <br /> - {% trans "Use \",\" to separate labels." %}
                     </p>
                 </li>
+                <li class="field comp-setting-entry">
+                    <div class="wrapper-comp-setting">
+                        <label for="openassessment_show_private_test_case_results_editor" class="setting-label">
+                            {% trans "Show Private Test Case Results"%}
+                        </label>
+                        <select id="openassessment_show_private_test_case_results_editor" class="input setting-input">
+                            <option value="0">{% trans "False"%}</option>
+                            <option value="1" {% if show_private_test_case_results %} selected="true" {% endif %}>
```

Comment:
what is this if condition achieving?

Reply:
It is reflecting the `show_private_test_case_results` value inside the block on the frontend.

It sets the default selected option to "True" if `show_private_test_case_results` is true in the block. Without this, we will always see "False" selected whenever the settings HTML renders.